dtos/http_request.go

```diff
@@ -0,0 +1,13 @@
+package dtos
+
+type CreateUpdateFormRequestDto struct {
+	Status  string  `json:"status"`
+	Content []Block `json:"content"`
+	OwnerId int64   `json:"ownerId"`
+}
+
+type CreateFormRequestDto struct {
+	Content      []Block `json:"content"`
+	ResponseById int64   `json:"responseById"`
```

Comment:
What is ResponsesById ?

Reply:
the id of the responder (user who submitted the form)